cli/commands/testdata/snake_request_body.json

```diff
@@ -11,13 +11,13 @@
         "hazardMap": "",
         "hazardMapAuthor": "",
         "royale": {
-          "shrinkEveryNTurns": 25
+          "shrinkEveryNTurns": 0
         },
         "squad": {
-          "allowBodyCollisions": true,
-          "sharedElimination": true,
-          "sharedHealth": true,
-          "sharedLength": true
+          "allowBodyCollisions": false,
+          "sharedElimination": false,
+          "sharedHealth": false,
+          "sharedLength": false
         }
```

Comment:
I think this is correct. The game type being tested is `standard`, so the squad settings don't make sense to be initialised. `false` makes sense as the zero values.

The reason that they used to be `true` was because the old code that created settings from ruleset was hard-coded to always set the squad settings to `true`.

I figured that the test was just written this way to make it work, but this makes more sense. Otherwise all the rulesets have to initialise all the settings.

Reply:
Actually, I take it back. It's okay to default these to `false` because I override them to `true` in the `defaultSettings` for the `play` command :)

Reply:
ah, that makes sense - thanks